gui/builtinViews/fittingView.py

```diff
@@ -492,10 +492,11 @@ def spawnMenu(self):
         while sel != -1 and sel not in self.blanks:
             mod = self.mods[self.GetItemData(sel)]
 
-            # Test if mod.isEmpty exists.
-            try:
-                mod.isEmpty
-            except AttributeError:
+            # Test if mod.isEmpty does not exist.
+            # Certain special module can be missing this trait
+            # Example: T3D modes
+            if not hasattr(mod, 'isEmpty'):
+                # Set it if missing, prevents later stack traces.
```

Comment:
Well, if we wanted to treat modes like a module it's required because it's used several places. vOv

Anyway, implemented `fittingMode` so now (at least for the purpose of context menus) modes are no longer items.